scripts/push_image_to_ecr.sh

```diff
@@ -113,13 +113,9 @@ tags=("$CIRCLE_SHA1" "$@")
 
 printf '%s\n' "${tags[@]}" | grep -q '^--' && error_usage "It looks like you're passing flags instead of tags"
 
-info "Fetching ECR login command"
-if ! command=$(aws ecr get-login --no-include-email --region "$region"); then
-    error "Problem executing 'aws ecr get-login command"
-fi
-
 info "Logging in to repo"
-$command || error "Problem logging in to repo"
+aws ecr get-login-password --region "$region" | docker login --username AWS --password-stdin "$repo_url" \
+    || error "Problem logging in to repo"
```

Comment:
Other option tested was to save the password to a var, but that threw a warning about not using `--password-stdin`:

```suggestion
info "Fetching ECR login password"
password=$(aws ecr get-login-password --region "$region") || error "Problem getting repo password"

 info "Logging in to repo"docker login --username AWS --password "$password" "$repo_url" || error "Problem logging in to repo"
```